packages/amplify_datastore/example/ios/amplify_datastore_unit_tests/ModelSchemaEquatableExtensions.swift

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Foundation
+import Amplify
+
+extension ModelSchema : Equatable {
+    public static func == (lhs: ModelSchema, rhs: ModelSchema) -> Bool {
+        return lhs.name == rhs.name && lhs.pluralName == rhs.pluralName && lhs.authRules == rhs.authRules && lhs.fields == rhs.fields && lhs.attributes == rhs.attributes
+    }
+}
+
+extension AuthRule : Equatable {
+    public static func == (lhs: AuthRule, rhs: AuthRule) -> Bool {
+        return lhs.allow == rhs.allow && lhs.ownerField == rhs.ownerField && lhs.identityClaim == rhs.identityClaim && lhs.groupClaim == rhs.groupClaim && lhs.groups == rhs.groups && lhs.groupsField == rhs.groupsField && lhs.operations == rhs.operations
+    }
+}
```

Comment:
can we wrap these lines? and other long ones?

Reply:
Sounds good, I went through and added line wraps

I looked through some of the model classes and added line wraps especially to the guard statements in init